Vagrantfile

```diff
@@ -1,3 +1,15 @@
 Vagrant.configure("2") do |config|
-  config.vm.box = "fedora/30-cloud-base"
+  config.vm.box = "dummy"
```

Comment:
Those changes shouldn't affect the ability to provision a DEV environment with virtualbox

Reply:
Hi @lioramilbaum What do you want me to do? a scrip with 2 different paths? 
Or one vagrant file with AWS credentials and the one you have provided here